gateway/src/apicast/cli/environment.lua

```diff
@@ -56,13 +56,17 @@ _M.default_environment = 'production'
 
 --- Default configuration.
 -- @tfield ?string ca_bundle path to CA store file
+-- @tfield ?string proxy_ssl_certificate path to SSL certificate
+-- @tfield ?string proxy_ssl_certificate_key path to SSL certificate key
 -- @tfield ?policy_chain policy_chain @{policy_chain} instance
 -- @tfield ?{string,...} nameservers list of nameservers
 -- @tfield ?string package.path path to load Lua files
 -- @tfield ?string package.cpath path to load libraries
 -- @table environment.default_config default configuration
 _M.default_config = {
     ca_bundle = resty_env.value('SSL_CERT_FILE'),
+    proxy_ssl_certificate = resty_env.value('SSL_PROXY_CERTIFICATE'),
+    proxy_ssl_certificate_key = resty_env.value('SSL_PROXY_CERTIFICATE_KEY'),
```

Comment:
We probably could name these like `APICAST_PROXY_*` (`APICAST_PROXY_CERTIFICATE` and `APICAST_PROXY_CERTIFICATE_KEY`). 

`APICAST_PROXY` is not the best name. Maybe `APICAST_UPSTREAM_HTTPS` ?